docs/target-setup-es2k.md

```diff
@@ -1,73 +1,163 @@
-# Install IPDK Networking Recipe
-IPDK infrap4d needs to be installed and run on the host natively. To install
-infrap4d and P4-SDE (components as per IPDK 23.07 release) individually, follow
-the instructions listed below. Note that, P4C is not required as this software
-includes P4C generated artifacts.
+# Target Setup for Intel IPU ES2K
+
+## Set Up Hardware Board
+Hardware setup requires the Intel IPU device to be connected to a link partner
+in a back-to-back manner. Refer to the setup topology in the
+FXP_P4_SDE_User_Guide.md included in the documentation in official CI release
+for details. This document also provides instructions on how to configure the
+machine with required BIOS settings, required third-party software, boot
+instructions, and system settings, as well as other information.
```

Comment:
Don't we need to mention how to stop the regular boot process by pressing 'N' and then copy the files?